### ISSUE TITLE
Document JSON file type for user/teams endpoints

### ISSUE THREAD

**Issue by jeremy-london** [DOMjudge/domjudge#879]
Currently the documentation does not provide examples on how to utilize the .json formData input for the `user/teams` API endpoints.

In the importJSON code I found that the JSON object must meet the following parameters for the function to work required this format:

```json
[
  {
    "group_ids": ["string"],
    "id": "number",
    "name": "string",
    "display_name": "string",
    "organization_id": "string"
  }
]
```

1. Currently `name` and `display_name` are required while the other params are optional due to a null condition if empty
2. `group_ids` selects the first index.. so why is this field an Array of strings?

**Comment by nickygerritsen** [comment]
The latter is because we are adhering to the [ICPC CCS spec](https://ccs-specs.icpc.io/contest_api#teams)

**Comment by thijskh** [comment]
According to that spec, display_name must not be required, so 1 is a buglet?

**Comment by nickygerritsen** [comment]
Seems like it yes.

**Comment by nickygerritsen** [comment]
Display name is optional already, since it is prefixed with `@` in the check.
Kinda ugly but OK for now.

We should still document this format. Also we might want to do a proper import, since this one requires you to add a file with the JSON in it. I'd say it makes more sense to have a `POST api/contests/x/teams` with the fields as body?

We do need to keep this version probably since we need it for CLICS compatibility.

**Comment by thijskh** [comment]
Fixed in  #919
